src/Raygun.Blazor/Models/StackTraceDetails.cs

```diff
@@ -94,8 +93,29 @@ internal StackTraceDetails(StackFrame frame)
             MethodToken = frame.GetMethod()?.MetadataToken;
         }
 
-        #endregion
+        /// <summary>
+        /// Creates a new instance of the <see cref="StackTraceDetails" /> class from a JavaScript stack trace frame.
+        /// Example: "causeErrors@http://localhost:5010/myfunctions.js:4:9"
+        /// </summary>
+        /// <param name="frame">JavaScript stack trace</param>
+        internal StackTraceDetails(string frame)
```

Comment:
This is the slowest way to do regex and you'll create a new RegEx instance for every line of every stack trace.

If you are going to use RegEx here you should REALLY consider using .NET 8 regex attributes to leverage codegen for optimal performance.

Reply:
Thanks! created an issue for it #31